src/app/view/components/dot-persona-selector/dot-persona-selector.component.ts

```diff
@@ -45,4 +53,10 @@ export class DotPersonaSelectorComponent implements OnInit {
     private setOptions(message: string, personas: DotPersona[]): DotPersona[] {
         return [{ name: message, identifier: '0' }, ...personas];
     }
+
+    private setDropdownWidth(personas: DotPersona[]): void {
+        const maxTextOption = _.maxBy(personas, (personaOption: DotPersona) => personaOption.name.length).name;
+        const dropdownSize = this.stringPixels.getTextWidth(maxTextOption) + this.arrowDropdownlComponentSize;
+        this.dropdownWidth = `${dropdownSize}px`;
+    }
```

Comment:
the same here

https://github.com/dotCMS/core-web/pull/673/files#diff-c6dc9be28b451d901acfd04e29d3f845R69